### PR TITLE
feat: provision rhmi cluster by default

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/INTLY-8422
https://issues.redhat.com/browse/SDA-2524

### CODE REVIEW

File: scripts/ocm/ocm.sh [hunk 1]
```diff
@@ -259,11 +258,19 @@ wait_for() {
...
+
+    api_url=$(jq .api.url < "${CLUSTER_DETAILS_FILE}")
     # Update cluster details (with master & console URL)
     ocm get "/api/clusters_mgmt/v1/clusters/${cluster_id}" | jq -r > "${CLUSTER_DETAILS_FILE}"
     # Create kubeconfig file & save admin credentials
     ocm get "/api/clusters_mgmt/v1/clusters/${cluster_id}/credentials" | jq -r .kubeconfig > "${CLUSTER_KUBECONFIG_FILE}"
-    ocm get "/api/clusters_mgmt/v1/clusters/${cluster_id}/credentials" | jq -r ".admin | .api_url = $(jq .api.url < "${CLUSTER_DETAILS_FILE}") | .console_url = $(jq .console.url < "${CLUSTER_DETAILS_FILE}")" > "${CLUSTER_CREDENTIALS_FILE}"
+
+    wait_for "oc --kubeconfig ${CLUSTER_KUBECONFIG_FILE} get route console -n openshift-console -o jsonpath='{.spec.host}' | grep -q console" "OpenShift console to be ready" "10m" "20"
+    console_url=\"https://$(oc --kubeconfig "${CLUSTER_KUBECONFIG_FILE}" get route console -n openshift-console -o jsonpath='{.spec.host}')\"
```
Comment (psturc) [inline scripts/ocm/ocm.sh:271]: for top-level RHMI installation, console URL cannot be fetched via ocm anymore.

File: scripts/ocm/ocm.sh [hunk 2]
```diff
@@ -92,7 +92,7 @@ create_cluster() {
 
     echo "Cluster ID: ${cluster_id}"
 
-    wait_for "ocm get /api/clusters_mgmt/v1/clusters/${cluster_id}/status | jq -r .state | grep -q ready" "cluster creation" "180m" "300"
+    wait_for "ocm get /api/clusters_mgmt/v1/clusters/${cluster_id} | jq -r .api.url | grep -q https://.*:6443" "cluster creation" "180m" "300"
```
Comment (psturc) [inline scripts/ocm/ocm.sh:95]: cluster should be ready once API URL is available.

**Comment by psturc** [comment]
@pawelpaszki thanks for review.

I was verifying this against **rhmi** ocm org. will merge this once **csqe** ocm organization allows creating top-level RHMI clusters.
(currently there's an issue with it and it's being discussed at the [slack channel](https://coreos.slack.com/archives/CB53T9ZHQ/p1594893907484400))

cc @maleck13 @matskiv

**Comment by matskiv** [comment]
LGTM
seems like a fix on OSD side was merged,
@psturc do you want to verify this with csqe account, or can I go ahead "/approve" it? :)

**Comment by psturc** [comment]
@matskiv thanks for looking into this. I tried it with csqe org account now and it still doesn't work. 

The conversation on the slack channel still continues...

**Comment by psturc** [comment]
Closing this PR
See https://issues.redhat.com/browse/INTLY-9595 for more details

**Pull request closed without merging**